extensions/blocks/eventbrite/save.js

```diff
@@ -57,15 +59,18 @@ function saveButton( eventId, attributes ) {
 		borderRadius: borderRadius ? borderRadius + 'px' : undefined,
 	};
 
+	// Saves link markup, but event handlers are added with inline javascript to prevent
+	// default link behavior (see the `jetpack_render_eventbrite_block` php function).
 	return (
-		<div>
+		<div className="wp-block-button">
 			<RichText.Content
-				id={ createWidgetId( eventId ) }
-				tagName="button"
 				className={ buttonClasses }
+				id={ createWidgetId( eventId ) }
+				role="button"
 				style={ buttonStyle }
+				tagName="a"
+				href={ url }
```

Comment:
Nice one @creativecoder, love that you got to remove the `noscript` with such a simple change! ✨